.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,60 @@
+## Communication
+
+We use [slack](https://goldcardcommunity.slack.com/) for talking about all `#website` matters.
```

Comment:
People will see this, but they still need to invited to our channel. Maybe this is a good thing as it builds visibility to the gold card program